gh-100689: Revert "bpo-41798: pyexpat: Allocate the expat_CAPI on the… (#100745)
python/cpython: Misc/NEWS.d/next/Library/2023-01-04-12-58-59.gh-issue-100689.Ce0ITG.rst

> Fix crash in ``_elementtree.c`` by reverting ``GH-24061``.

Reviewer: ```suggestion
Fix crash in ``_elementtree.c`` by statically allocating ``PyExpat_CAPI`` capsule. 
```

> Fix crash in ``pyexpat`` by statically allocating ``PyExpat_CAPI`` capsule.

Reviewer: ```suggestion
Fix crash in :mod:`pyexpat` by statically allocating ``PyExpat_CAPI`` capsule.
```

> Fix crash in ``_elementtree.c`` by statically allocating ``PyExpat_CAPI`` capsule.

Reviewer 2: I'd use the user visible module name instead of the source file name; the changelog is for users, not core devs :)

> Fix crash in ``_elementtree.c`` by statically allocating ``PyExpat_CAPI`` capsule. 

Reviewer: ```suggestion
Fix crash in ``_elementtree.c`` by statically allocating ``PyExpat_CAPI`` capsule.
```
